packages/react-devtools-shared/src/devtools/views/DevTools.js

```diff
@@ -127,6 +128,15 @@ export default function DevTools({
     [enabledInspectedElementContextMenu, viewAttributeSourceFunction],
   );
 
+  const switchTabKeys = ['ctrl+1', 'ctrl+2', 'command+1', 'command+2'];
+  Mousetrap.bind(switchTabKeys, (e, keyCombination) => {
```

Comment:
This is a side effect, so it would need to be inside of a `useEffect`:
```js
useEffect(() => {
  if (showTabBar) {
    Mousetrap.bind(...)
  }
}, [showTabBar]);